### PR TITLE
Feat/booking flight

### PR DESCRIPTION
**Pull request by ImMPrada**
This PR:

- updates the models, following this, removing BookingFlights and a Flight will have many bookings:
   image

- enables Fligth#Index to choose a number of passengers:
  image

- enables to book a flight
   image

- adds a service `ModelsServices::Creators::Bookings` to manage the creation of a Booking by adding and verifying the validity of passenger's information

### CODE REVIEW

File: app/services/models_services/creators/bookings.rb [hunk 1]
```diff
@@ -0,0 +1,81 @@
+module ModelsServices
+  module Creators
+    class Bookings
```
Comment (mapra99) [inline app/services/models_services/creators/bookings.rb]: A namespace called `ModelsServices` inside the servces folder feels very weird. What makes a service class be inside or outside ModelsService?

Services are usually meant to be there to store business logic. I think it's better if you think of namespaces inside services as a way to separate business domains that can stand on their own, so it's better if you group classes in namespaces according to their domain, not what they do. Hence, I think it's better if you have a `Bookings` namespace that will contain everything related to the bookings business domain, such as creation, confirmation, modification, cancelation, etc. So as your application grows, you can just keep adding more namespaces as business domains grow, for example, a separate `Checkout` or `Authentication` service. Defining this correctly in real apps is part of defining the backend architecture, and by doing it this way you can have namespaces grouping classes that are more related to each other and more independent from other namespaces.

So this class could be `Bookings::Creator` or `Bookings::BookingCreator`

File: app/services/models_services/creators/bookings.rb [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+      attr_writer :amount_of_passengers
+
+      def self.whit_just_amount_of_passengers(flight:, user:, amount_of_passengers:)
```
Comment (mapra99) [inline app/services/models_services/creators/bookings.rb]:
```suggestion
      def self.with_just_amount_of_passengers(flight:, user:, amount_of_passengers:)
```

**Review by mapra99 — APPROVED** [review]

File: app/services/models_services/creators/bookings.rb (hunk 1 above)
Comment (ImMPrada) [reply app/services/models_services/creators/bookings.rb]: yeah, I understand, so lets go with `Bookings::Creator`

File: app/services/models_services/creators/bookings.rb (hunk 2 above)
Comment (ImMPrada) [reply app/services/models_services/creators/bookings.rb]: fixed!

File: app/services/models_services/creators/bookings.rb (hunk 1 above)
Comment (ImMPrada) [reply app/services/models_services/creators/bookings.rb]: Done!

**Pull request merged**